srcCpp/Infrastructure_micro.h

```diff
@@ -54,6 +60,10 @@ class PerftestClock {
     RTI_INT32 clockSec;
     RTI_UINT32 clockUsec;
 
+#if defined(RTI_WIN32)
+    double _frequency = 0.0;
```

Comment:
I think you can skip having clockSec and clockUsec:

#ifndef ...


#else 

double _fequency = 0.0;